test/integration/aggregateQuery.spec.ts

```diff
@@ -43,220 +41,179 @@ describe("NodeJS Aggregate Query Tests", async function() {
+        expected: testdata.sum / testdata.numberOfDocumentsWithNumbericId,
+        condition: `IS_NUMBER(r.${partitionKey})`
+      },
+      {
+        operator: "COUNT",
+        expected: testdata.numberOfDocuments,
+        condition: "true"
+      },
+      { operator: "MAX", expected: "xyz", condition: "true" },
+      { operator: "MIN", expected: null, condition: "true" },
+      {
+        operator: "SUM",
+        expected: testdata.sum,
+        condition: `IS_NUMBER(r.${partitionKey})`
+      }
+    ];
 
-      aggregateConfigs.forEach(function(config) {
-        let query = util.format(aggregateQueryFormat, config.operator, partitionKey, config.condition);
-        let testName = util.format("%s %s", config.operator, config.condition);
-        testConfigs.push({
-          testName,
-          query,
-          expected: config.expected
-        });
+    aggregateConfigs.forEach(function({ operator, condition, expected }) {
+      let query = `SELECT VALUE ${operator}(r.${partitionKey}) FROM r WHERE ${condition}`;
+      let testName = `${operator} ${condition}`;
 
-        query = util.format(aggregateOrderByQueryFormat, config.operator, partitionKey, config.condition, partitionKey);
-        testName = util.format("%s %s OrderBy", config.operator, config.condition);
-        testConfigs.push({
-          testName,
-          query,
-          expected: config.expected
-        });
+      testConfigs.push({
+        testName,
+        query,
+        expected
       });
 
-      const aggregateSinglePartitionQueryFormat = "SELECT VALUE %s(r.%s) FROM r WHERE r.%s = '%s'";
-      const aggregateSinglePartitionQueryFormatSelect = "SELECT %s(r.%s) FROM r WHERE r.%s = '%s'";
-      const samePartitionSum =
-        (testdata.numberOfDocsWithSamePartitionKey * (testdata.numberOfDocsWithSamePartitionKey + 1)) / 2.0;
-      const aggregateSinglePartitionConfigs = [
-        {
-          operator: "AVG",
-          expected: samePartitionSum / testdata.numberOfDocsWithSamePartitionKey
-        },
-        {
-          operator: "COUNT",
-          expected: testdata.numberOfDocsWithSamePartitionKey
-        },
-        {
-          operator: "MAX",
-          expected: testdata.numberOfDocsWithSamePartitionKey
-        },
-        { operator: "MIN", expected: 1 },
-        { operator: "SUM", expected: samePartitionSum }
-      ];
-
-      aggregateSinglePartitionConfigs.forEach(function(config) {
-        let query = util.format(
-          aggregateSinglePartitionQueryFormat,
-          config.operator,
-          testdata.field,
-          partitionKey,
-          uniquePartitionKey
-        );
-        let testName = util.format("%s SinglePartition %s", config.operator, "SELECT VALUE");
-        testConfigs.push({
-          testName,
-          query,
-          expected: config.expected
-        });
+      query = `SELECT VALUE ${operator}(r.${partitionKey}) FROM r WHERE ${condition} ORDER BY r.${partitionKey}`;
```

Comment:
Can we test that aggregates without the `VALUE`. If it's v3 SDK they should be blocked. If v2 SDK they should work in a single physical partition to avoid a breaking change.

Reply:
Yep. I have a test for this at the bottom of the file. They are blocked.